tests/_serialization/_data.py

```diff
@@ -86,6 +86,5 @@ def _deserialize(self, value):
     (properties.Boolean, 1.0),
     (properties.Boolean, 'asdf'),
     (properties.UUID, 'wrong-number-of-chars'),
-    (properties.UUID, uuid.uuid4()),
```

Comment:
Modified to be valid, since deserialization routines aren't bothered by previously deserialized values anymore.